test/collectors_test.lua

```diff
@@ -40,29 +38,30 @@ g.test_counter = function()
     local collectors = metrics.collectors()
     local observations = metrics.collect()
     local obs = utils.find_obs('cnt', {}, observations)
-    t.assertEquals(#collectors, 1, 'counter seen as only collector')
-    t.assertEquals(obs.value, 8, '3 + 5 = 8 (via metrics.collectors())')
+    t.assert_equals(#collectors, 1, 'counter seen as only collector')
+    t.assert_equals(obs.value, 8, '3 + 5 = 8 (via metrics.collectors())')
 
-    t.assertEquals(c:collect()[1].value, 8, '3 + 5 = 8')
+    t.assert_equals(c:collect()[1].value, 8, '3 + 5 = 8')
 
-    ensure_throws('counter:inc(<negative value>)', function()
+    t.assert_error_msg_contains("Counter increment should not be negative", function()
         c:inc(-1)
     end)
-    ensure_throws('counter:dec(...)', function()
+
+    t.assert_error_msg_contains("attempt to call method 'dec' (a nil value)", function()
```

Comment:
counter.dec == nil should be enough. Why do you call it?

Reply:
Originally there was 
```
ensure_throws('counter:dec(...)', function()
        c:dec(1)
end)
```
and I simply rewrote it. Replaced with `assert_equals(..., nil, ...)`.